### PR TITLE
Correct Typo and correct exception name in JavaScript instrumentation…

### PR DESCRIPTION
**Pull request by AvasDream**
Hi All, 

I stumbled upon two Typos while reading the Documentation. The first is in line 189 (`lifetyime`) and the second is in line 265 (`catch (eexrr)`).  

---

Edit(@chalin)

Preview: https://deploy-preview-1230--opentelemetry.netlify.app/docs/instrumentation/js/instrumentation/

### CODE REVIEW

**Comment by chalin** [comment]
Thanks for the PR @AvasDream. You'll need to sign the CLA before we proceed.

**Comment by AvasDream** [comment]
@chalin I have signed the CLA :)

**Review by chalin — COMMENTED** [review]
I would have preferred separate PRs for the fixes vs the reformatting of code-excerpts, since the former is clearly desirable (and I would have merged right away), whereas the latter is subjective and open to debate. By combining these two in the same PR it also makes it more difficult to see what the non-formatting related changes are. Might you consider splitting these?

File: content/en/docs/instrumentation/js/instrumentation.md [hunk 1]
```diff
@@ -262,9 +287,8 @@ It can be a good idea to record exceptions when they happen. It's recommended to
 ```js
 try {
   doWork();
-} catch (eexrr) {
-  span.recordException(ex),
-  span.setStatus({ code: otel.SpanStatusCode.ERROR })
+} catch (ex) {
+  span.recordException(ex), span.setStatus({ code: otel.SpanStatusCode.ERROR });
```
Comment (legendecas) [inline content/en/docs/instrumentation/js/instrumentation.md:291]: I think the statement trailing comma is a kinda typo for semicolons. It is unusual for us to use the comma operator for two distinct expressions.

File: content/en/docs/instrumentation/js/instrumentation.md [hunk 2]
```diff
@@ -186,23 +208,23 @@ function doWork(parent) {
...
 ```
 
 You can also add an object with more data to go along with the message:
 
 ```js
-span.addEvent('some log', {
-  'log.severity': 'error',
-  'log.message': 'Data not found',
-  'request.id': requestId,
+span.addEvent("some log", {
```
Comment (legendecas) [inline content/en/docs/instrumentation/js/instrumentation.md:224]: In otel-js repo, we use the single quote marks for string literals.

**Review by legendecas — COMMENTED** [review]
I'm fine with most of the format except two nits.

**Comment by chalin** [comment]
Thanks for the feedback @legendecas.
@AvasDream - can you make the changes?

File: content/en/docs/instrumentation/js/instrumentation.md [hunk 3]
```diff
@@ -10,21 +10,21 @@ This guide will cover creating and annotating spans, creating and annotating met
...
   doWork();
 }
 
 function doWork() {
-  console.log("work...")
+  console.log("work...");
   // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {
-  }
+  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
```
Comment (blumamir) [inline content/en/docs/instrumentation/js/instrumentation.md:22]: not related to this PR which fixes typos, but the line `Math.floor(Math.random() * 40000000)` will be re-evaluated again and again for each iteration which is probably not what was intended

File: content/en/docs/instrumentation/js/instrumentation.md (hunk 1 above)
Comment (dyladan) [reply content/en/docs/instrumentation/js/instrumentation.md:291]:
```suggestion
  span.recordException(ex);
  span.setStatus({ code: otel.SpanStatusCode.ERROR });
```

File: content/en/docs/instrumentation/js/instrumentation.md (hunk 2 above)
Comment (chalin) [reply content/en/docs/instrumentation/js/instrumentation.md]: Fixed

**Pull request merged**